friends/src/components/Friends.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import './Friends.css';
+
+class Friends extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      friends: []
+    };
+  }
+
+  deleteFriend = (e, id) => {
+    e.preventDefault();
+    this.props.deleteFriend(id);
+  };
+
+  updateForm = (e, id) => {
+    e.preventDefault();
+    this.props.setUpdateForm(id);
```

Comment:
Line 19 invokes setUpdateForm from App.js and passes it an ID and sets activeItem to the ID.